### PR TITLE
[DEV-2275] add mail table

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/subdomains/supporting/notification/entities/mail.entity.ts
Comment (davidleomay) [inline src/subdomains/supporting/notification/entities/mail.entity.ts:1]: Can we combine it with the existing notification entity?

File: src/subdomains/supporting/notification/services/notification-job.service.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      const isSuppressed = await this.notificationService.isSuppressed(mail);
+      if (isSuppressed) return;
+
+      try {
+        await this.mailService.send(mail);
+
+        await this.notificationService.updateNotification(notification, { isComplete: true });
+      } catch (e) {
+        this.logger.error('Error in sendMail', e);
```
Comment (davidleomay) [inline src/subdomains/supporting/notification/services/notification-job.service.ts:59]: Maybe we should have a different message here than for the initial send?
`'Error during mail send retry:', e`?

**Comment by Yannick1712** [comment]
Add migration

**Review by davidleomay — APPROVED** [review]
Add migration

File: src/subdomains/generic/user/models/account-merge/account-merge.service.ts [hunk 2]
```diff
@@ -57,6 +58,8 @@ export class AccountMergeService {
         ],
         suffix: [{ key: MailKey.SPACE, params: { value: '4' } }, { key: MailKey.DFX_TEAM_CLOSING }],
       },
+      options: { debounce: 60000 },
+      metadata: { context: MailContext.ACCOUNT_MERGE_REQUEST, correlationId: `AccountMerge&${master.id}&${slave.id}` },
```
Comment (davidleomay) [inline src/subdomains/generic/user/models/account-merge/account-merge.service.ts:62]: correlationId: `${request.id}`

**Review by davidleomay — APPROVED** [review]

**Pull request merged**